lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -89,13 +94,77 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 		subProto.NetworkId = &protos.NetworkID{Id: networkID}
 		subProtos = append(subProtos, subProto)
 	}
+
+	// Short-circuit subscriber data transmission by comparing cloud & gateway digests.
```

Comment:
Nit: prefer "and" over using ampersands. Rationale is (a) it looks like the bitwise-and character, and (b) stdlib Go doesn't use it almost at all
```suggestion
	// Short-circuit subscriber data transmission by comparing cloud and gateway digests.
```